File: Java/eclipse-workspace/Prj12_Pokemon/src/entities/Pokemon.java
package entities;

public class Pokemon {

	private int id;
	private String name;
	private String type1;
	private String type2;
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getType1() {
		return type1;
	}
	public void setType1(String type1) {
		this.type1 = type1;
	}
	public String getType2() {
		return type2;
	}
	public void setType2(String type2) {
		this.type2 = type2;
	}
	@Override
	public String toString() {
		return "Pokemon [id=" + id + ", name=" + name + ", type1=" + type1 + ", type2=" + type2 + "]";
	}
	
	
	
	
	
}

File: Php/ISD2025/L07/app/model/Pokemon.php
<?php 
class Pokemon{
    private $id;
    private $name;
    private $immagine;

    public function __get($name) {
        if(isset($this->$name)) 
        return $this->$name;
    }
    public function __set($name, $value) {
        $this->$name = $value;
    }
    public function __tostring() {
        return "". $this->id ."". $this->name ;
    }
}